rego/rules/arm/postgresql/enforce_ssl.rego

```diff
@@ -35,22 +35,9 @@ __rego__metadoc__ := {
 
 input_type = "arm"
 
-resource_type = "MULTIPLE"
+resource_type = "Microsoft.DBforMySQL/servers"
```

Comment:
Should this be:
```suggestion
resource_type = "Microsoft.DBforPostgreSQL/servers"
```

Reply:
Yes, thanks for the fix!  Annoying the tests don't catch this.  Tested using `regula run`.